etl/steps/data/garden/wb/2023-11-21/worldwide_bureaucracy_indicators.py

```diff
@@ -0,0 +1,110 @@
+"""Load a meadow dataset and create a garden dataset."""
+
+
+from owid.catalog import Table
+
+from etl.data_helpers import geo
+from etl.helpers import PathFinder, create_dataset
+
+# Get paths and naming conventions for current step.
+paths = PathFinder(__file__)
+
+
+def run(dest_dir: str) -> None:
+    #
+    # Load inputs.
+    #
+    # Load meadow dataset.
+    ds_meadow = paths.load_dataset("worldwide_bureaucracy_indicators")
+
+    # Read table from meadow dataset.
+    tb = ds_meadow["worldwide_bureaucracy_indicators"].reset_index()
+
+    #
+    # Process data.
+
+    tb, indicator_dict = reformat_table(tb)
+
+    tb = geo.harmonize_countries(
+        df=tb,
+        countries_file=paths.country_mapping_path,
+    )
+    tb = tb.set_index(["country", "year"], verify_integrity=True)
+
+    #
+    # Save outputs.
+
+    tb = add_metadata_from_dict(tb, indicator_dict)
+
+    # Create a new garden dataset with the same metadata as the meadow dataset.
+    ds_garden = create_dataset(
+        dest_dir, tables=[tb], check_variables_metadata=True, default_metadata=ds_meadow.metadata
+    )
+
+    # Save changes in the new garden dataset.
+    ds_garden.save()
+
+
+def reformat_table(tb: Table) -> [Table, dict]:
```

Comment:
I think this needs to be: 

`def reformat_table(tb: Table) -> tuple[Table, dict]:`

to pass CI